### DISCUSSION TITLE
Calling for feedbacks or ideas to further improve the DApp Bootcamp!

### DISCUSSION THREAD

**Discussion by zxstim** [buildstationorg/evm-dapp-bootcamp#2] [Ideas]
For all bootcamp participants, please comment any ideas or feedbacks that you have for our team to continuously improve the bootcamp

**Comment by lamdanghoang** [comment]
I want to know more about ERC20, ERC721.
Some cases related to smart contract vulnerabilities: How to check & solution.
Please consider about that. Thank you.

**Reply by zxstim** [reply]
Sounds good. Will update ERC20 and ERC721 contracts inside the repo.
